flask/app/routes.py

```diff
@@ -162,3 +171,61 @@ def change_password():
     except:
         db.session.rollback()
         return 'Server error', 500
+
+@app.route('/api/participant', methods=['GET'], endpoint='partipants_list')
+def partipants_list():
+    db_participants = db.session.query(models.Participant).all()
+    participants = [
+        {
+            'family_codename': participant.family_id,
+            'participant_codename': participant.participant_codename,
+            'sex': participant.sex,
+            'participant_type': participant.participant_type,
+            'affected': participant.affected,
+            'solved': participant.solved,
+            'created': participant.created,
+            'created_by': participant.created_by,
+            'updated': participant.updated_by,
+            'tissue_samples': [tissue for tissue in participant.tissue_samples.all()]
```

Comment:
My only thought here is that maybe we should show the datasets of a participant instead of (or in addition to) tissue_samples. Unless there was a reason it's left out that I'm forgetting?

Reply:
That would make sense, should I add that into the participant model?

Reply:
Would it be possible to do something like `'datasets':` [tissue.datasets.all() for tissue in participant.tissue_samples.all()]`? I've got another branch checked out and have loaded up the legacy database in it so can't test this myself super easily, but it looks like there's a link to the datasets via the TissueSample table.

Reply:
That works! I just need to figure out how to serialize the tissue object.